app.test.js

```diff
@@ -46,4 +46,45 @@ describe('getCategories', () => {
             })
           });
       });
-})
\ No newline at end of file
+})
+
+describe('getReviews', () => {
+  test("Responds with an object containing an array of review objects", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({body}) => {
+          expect(body.reviews).toEqual(expect.any(Array));
```

Comment:
This first test is unnecessary. Checking the length, and iterating through with forEach to check properties means it is already proven to be an array in the next test.